src/dotnet/Fable.Core/Fable.Core.fsproj

```diff
@@ -21,6 +21,6 @@
     <Content Include="..\..\..\build\fable-core\" PackagePath="fable-core\" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="FSharp.Core" Version="4.2.1" />
+    <PackageReference Include="FSharp.Core" Version="4.2.3" />
```

Comment:
I was trying to reduce a bit the requirement for libraries depending on Fable.Core, but I guess it doesn't make too much of a difference :)